### DISCUSSION TITLE
Detect bash commands and coloring them in SSH

### DISCUSSION THREAD

**Discussion by josepcorrea** [Eugeny/tabby#6442] [General]
Is it possible for the terminal to detect bash commands and color them?

![image](https://user-images.githubusercontent.com/105700525/168744680-86850399-166e-43ed-a3a1-597a2c0ef9c6.png)

**Comment by Eugeny** [comment]
This is generally a shell's job. While I'm not aware of a syntax highlighter for bash, there's a ZSH plugin that does this: https://github.com/zsh-users/zsh-syntax-highlighting

**Reply by josepcorrea** [reply]
thank you very much
